src/Elm/Parser/Patterns.elm

```diff
@@ -125,16 +125,14 @@ qualifiedPatternArg =
 
 allPattern : Parser state (Node Pattern)
 allPattern =
-    Core.succeed AllPattern
-        |. Core.symbol "_"
+    Core.map (\() -> AllPattern) (Core.symbol "_")
         |> Node.parserCore
         |> Combine.fromCore
 
 
 unitPattern : Parser state (Node Pattern)
 unitPattern =
-    Core.succeed UnitPattern
-        |. Core.symbol "()"
+    Core.map (\() -> UnitPattern) (Core.symbol "()")
```

Comment:
Is this faster?

Reply:
This seemed to be the default in the rest of the codebase. I imagine this is faster because it's more direct (instead of construct then andMap via |=)

Reply:
Here's the result (ran twice, always about 10%):
![image](https://github.com/user-attachments/assets/117f2568-bad5-46cb-90d3-2c366a20c3ad)

Reply:
Good to know!